### PR TITLE
renamed ObservableObject file name and updated gitignore

### PR DESCRIPTION
**Pull request by manojmahapatra**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,6 @@
 xcuserdata/
 
 Carthage/Build/
+
+.DS_Store
+Sources/.DS_Store
```
Comment (ddddxxx) [inline .gitignore]: A `.DS_Store` is enough. Also it would be better adding a newline at end of the file.

Comment (manojmahapatra) [reply .gitignore]: Thanks for the catch. updated now.

**Review by ddddxxx — APPROVED** [review]

**Comment by manojmahapatra** [comment]
Thank you @ddddxxx

**Comment by ddddxxx** [comment]
Thank you for your contribution!

**Pull request merged**